go.mod

```diff
@@ -0,0 +1,33 @@
+module github.com/google/keytransparency
+
+go 1.12
+
+require (
+	github.com/benlaurie/objecthash v0.0.0-20180202135721-d1e3d6079fc1
+	github.com/go-sql-driver/mysql v1.4.1 // indirect
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/golang/mock v1.3.1
+	github.com/golang/protobuf v1.3.1
+	github.com/google/btree v1.0.0 // indirect
+	github.com/google/certificate-transparency-go v1.0.21 // indirect
```

Comment:
This is [a fairly old release of certificate-transparency-go](https://github.com/google/certificate-transparency-go/releases/tag/v1.0.21) (2018-08-20) compared to the version of Trillian being used (from 2019-05-24). You're certain this won't cause any problems?

Reply:
It is old... but it is also the latest tag.  Should we create a new release of ct-go?

Reply:
It's not causing any issues the I am aware of

Reply:
It's most likely fine, but this combination of versions is unlikely to have been tested beyond the unit/integration tests (compared to picking commits/releases that are closer together temporally, which will be more similar to versions that went through CI and into production together).